ngraph_bridge/ngraph_encapsulate_op_utils.h

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright 2017-2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#ifndef NGRAPH_TF_BRIDGE_GET_PIPELINED_TENSORS_H
+#define NGRAPH_TF_BRIDGE_GET_PIPELINED_TENSORS_H
+
+#pragma once
+
+#include "tensorflow/core/graph/graph.h"
+
+#include "logging/ngraph_log.h"
+#include "ngraph_bridge/ngraph_pipelined_tensors.h"
+#include "ngraph_bridge/ngraph_tensor_manager.h"
+
+using namespace std;
+namespace tensorflow {
+
+namespace ngraph_bridge {
+
```

Comment:
Please add some comments here about what this function does

Reply:
done